task4.py

```diff
@@ -0,0 +1,9 @@
+omnivores = {'John': 'vegetarian', 'Michael': 'omnivore', 'Inga': 'vegetarian', 'Oliver': 'omnivore'}
+vegetarians = {'Sam': 'vegetarian', 'Kat': 'vegetarian'}
+
+omnivores_and_vegetables = {**omnivores, **vegetarians}
+
+for key, values in omnivores_and_vegetables.items():
+    if values == 'vegetarian':
```

Comment:
actually, if you eat meat, you can eat vegetables as well. So the task is to display all persons from both sets of data